src/routes/webhooks/customerio.ts

```diff
@@ -141,22 +162,14 @@ export const customerio = async (fastify: FastifyInstance): Promise<void> => {
       }
 
       const payload = req.body;
-      const userId = payload.data.identifiers.id;
-      // Delete personal data
-      delete payload.data.identifiers;
-      delete payload.data.recipient;
-
-      const event = {
-        event_timestamp: new Date(payload.timestamp * 1000),
-        event_id: payload.event_id,
-        session_id: payload.event_id,
-        visit_id: payload.event_id,
-        user_id: userId,
-        event_name: `${payload.object_type} ${payload.metric}`,
-        app_platform: 'customerio',
-        extra: JSON.stringify(payload.data),
-      };
-      await sendAnalyticsEvent([event]);
+
+      // There are multiple events that should trigger the subscription sync
+      if (payload.metric.includes('subscri')) {
```

Comment:
Feels error prone, do we control the metric values?

Reply:
nope, do you rather see hard coded values? i'm just afraid to miss some :triggered:

Reply:
I would, but I also don't know how many of those is there.

Reply:
done